client/layout/masterbar/masterbar-cart-button.tsx

```diff
@@ -0,0 +1,64 @@
+import { Popover } from '@automattic/components';
+import { CheckoutErrorBoundary } from '@automattic/composite-checkout';
+import { MiniCart } from '@automattic/mini-cart';
+import { useShoppingCart } from '@automattic/shopping-cart';
+import { useTranslate } from 'i18n-calypso';
+import { useRef, useState } from 'react';
+import MasterbarItem from './item';
+
+import './masterbar-cart-button-style.scss';
+
+export type MasterbarCartButtonProps = {
+	selectedSiteSlug: string | undefined;
+	goToCheckout: ( siteSlug: string ) => void;
+};
+
+export function MasterbarCartButton( {
+	selectedSiteSlug,
+	goToCheckout,
+}: MasterbarCartButtonProps ): JSX.Element | null {
+	const { responseCart, reloadFromServer } = useShoppingCart( selectedSiteSlug );
+	const masterbarButtonRef = useRef( null );
+	const [ isActive, setIsActive ] = useState( false );
+	const translate = useTranslate();
+
+	if ( ! selectedSiteSlug || responseCart.products.length < 1 ) {
+		return null;
+	}
+
+	const onClick = () => {
+		setIsActive( ( active ) => {
+			if ( ! active ) {
+				reloadFromServer();
+			}
+			return ! active;
+		} );
+	};
+	const onClose = () => setIsActive( false );
+	const tooltip = translate( 'View my Shopping Cart' );
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **35** times:
`translate( 'My shopping cart' )` *ES Score: 7*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/calypso-masterbar-cart)